app/facades/businesses_search_facade.rb

```diff
@@ -1,7 +1,18 @@
 class BusinessesSearchFacade
   def self.utilities_search(location, type)
-    utilities = BusinessesSearchService.utility_search(location, type)
-    open_businesses = utilities[:data].select {|result| result[:is_closed] == false}
-    open_businesses.map{|result| Utility.new(result)}
+    businesses = BusinessesSearchService.utility_search(location, type)
+    is_open(businesses)
+  end
+
+  def self.recreation_search(location, type)
+    businesses = BusinessesSearchService.recreation_search(location, type)
+    is_open(businesses)
+  end
+
```

Comment:
Would this be able to be refactored into a dynamic search method, since the 'type' parameter is the main signifier on what business type is generated?

Reply:
I don't think so, because the service needs the specific search term for the microservice, which couldn't easily be passed through. I think this is a good way to keep some degree of separation between between all of our dynamic routes